### PR TITLE
Ship Select now clearly and visibly shows what faction a ship is on both purchase and late join.

### PR DESCRIPTION
**Pull request by ZephyrTFA**

### CODE REVIEW

File: code/__HELPERS/names.dm [hunk 1]
```diff
@@ -235,3 +235,39 @@ GLOBAL_DATUM(syndicate_code_response_regex, /regex)
 				. += "."
 			else
 				. += ", "
+
+/// List of all ship factions to their prefixes.
+GLOBAL_LIST_INIT(ship_faction_to_prefixes, list(
+	"Syndicate" = list(
+		"SEV",
+		"SSV",
+		"SGSV",
```
Comment (Erikafox) [inline code/__HELPERS/names.dm:246]: SGSV is solgov

File: code/__HELPERS/names.dm [hunk 2]
```diff
@@ -235,3 +235,39 @@ GLOBAL_DATUM(syndicate_code_response_regex, /regex)
...
+		"SRSV",
```
Comment (Erikafox) [inline code/__HELPERS/names.dm:249]: SRSV is Saint Romain

File: code/__HELPERS/names.dm [hunk 3]
```diff
@@ -235,3 +235,39 @@ GLOBAL_DATUM(syndicate_code_response_regex, /regex)
...
+		"SV",
+		"IMV",
+		"ISV",
+	),
+	"Inteq Risk Management Group" = list(
+		"IRMV",
+	),
+	"Colonial Minutemen" = list(
+		"CMSV",
+		"CMGSV",
```
Comment (Erikafox) [inline code/__HELPERS/names.dm:261]: +CMBSV

**Review by Erikafox — CHANGES_REQUESTED** [review]

File: code/__HELPERS/names.dm [hunk 4]
```diff
@@ -235,3 +235,39 @@ GLOBAL_DATUM(syndicate_code_response_regex, /regex)
...
 				. += ", "
+
+/// List of all ship factions to their prefixes.
+GLOBAL_LIST_INIT(ship_faction_to_prefixes, list(
+	"Syndicate" = list(
+		"SEV",
+		"SSV",
+		"SGSV",
+		"SRSV",
+		"STSV",
```
Comment (Erikafox) [inline code/__HELPERS/names.dm]: Also what the fuck uses STSV.

File: code/__HELPERS/names.dm (hunk 3 above)
Comment (ZephyrTFA) [reply code/__HELPERS/names.dm:261]: not in code or jsons anywhere, will not be included.

**Review by Erikafox — APPROVED** [review]

**Review by MarkSuckerberg — APPROVED** [review]
this will be done FAR better in the future. i don't like this but it'll work

**Pull request merged**